modules/identity-shiro/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -0,0 +1,11 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+grant {
+
```

Comment:
I'm not sure, but you might have to add jackson-databind permissions here to prevent access relate exceptions

Reply:
All the code is exercised and does not need any additional policy grants, as needed we can look into adding them